app/services/usps_confirmation_maker.rb

```diff
@@ -42,4 +43,8 @@ def generate_otp
     # Crockford encoding is 5 bits per character
     Base32::Crockford.encode(SecureRandom.random_number(2**(5 * 10)), length: 10)
   end
+
+  def update_proofing_cost
+    Db::ProofingCost::AddUserProofingCost.call(profile.user.id, :gpo_letter)
```

Comment:
only incur proofing cost when we know definitively that we've updated the gpo tables for batch uploading